ui/src/table.tsx

```diff
@@ -943,16 +943,22 @@ export const
     }), [filteredItems, groups, expandedRefs, isMultiple, items, m, onFilterChange, selectedFilters, selection, sort, setFiltersInBulk])
 
     return (
-      <div data-test={m.name} style={{ position: 'relative', height: computeHeight() }}>
-        <Fluent.Stack horizontal horizontalAlign='space-between' verticalAlign='end'>
-          {groupable && !m.pagination && <Fluent.Dropdown data-test='groupby' label='Group by' selectedKey={groupByKey} onChange={onGroupByChange} options={groupByOptions} styles={{ root: { width: 300 } }} />}
-          {!!searchableKeys.length && <Fluent.SearchBox data-test='search' placeholder='Search' onChange={onSearchChange} value={searchStr} styles={{ root: { width: '50%', maxWidth: 500 } }} />}
-        </Fluent.Stack >
+      <div data-test={m.name} style={{ position: 'relative', height: computeHeight()  }}>
+        <Fluent.Stack horizontal horizontalAlign='space-between' verticalAlign="end"  >
```

Comment:
Is the `verticalAlign` necessary here?

Reply:
Not sure, I think it caused some of the issues I had.